cmake/std/atdm/cts1/environment.sh

```diff
@@ -33,3 +33,6 @@ export ATDM_CONFIG_SLURM_DEFAULT_ACCOUNT=fy150090
 source $ATDM_SCRIPT_DIR/common/toss3/environment_new.sh
 
 export ATDM_CONFIG_TRIL_CMAKE_INSTALL_PREFIX_DATE_BASE_DEFAULT=/projects/atdm_devops/trilinos_installs/
+export ATDM_CONFIG_MAKE_INSTALL_GROUP_DEFAULT=wg-run-as-atdm-devops
+
+export ATDM_CONFIG_COMPLETED_ENV_SETUP=TRUE
```

Comment:
This is already set by https://github.com/trilinos/Trilinos/pull/7285/files#diff-34f163da6699ffe6dc9b9bca2b421e78R33, could that cause a data race?